### ISSUE TITLE
flow-typed v2.2.0 fails on windows

### ISSUE THREAD

**Issue by Awpteamoose** [flowtype/flow-typed#1313]
```
npx flow-typed
...\node_modules\flow-typed\dist\cli.js
Invalid regular expression: /definitions\npm\(@[^\]*\)?[^\]*\?/: Unterminated character class
```

It doesn't matter what commands I run and what folder I run them in - end result is the same.

* node v8.1.2
* Windows 10 x64

**Comment by kulmajaba** [comment]
Same issue for me on Windows.

The issue is in \dist\commands\runTests.js line 70:

    var basePathRegex = new RegExp('definitions' + _node.path.sep + 'npm' + _node.path.sep + '(@[^' + _node.path.sep + ']*' + _node.path.sep + ')?[^' + _node.path.sep + ']*' + _node.path.sep + '?');

On Windows, `_node.path.sep` is `\` but it's also the escape symbol, so the symbols have to be double-escaped: `\\\\`

Another option is to use `/` for Windows as well. From [Node.js docs](https://nodejs.org/api/path.html#path_path_sep): 

> Note: On Windows, both the forward slash (/) and backward slash (\) are accepted as path segment separators; however, the path methods only add backward slashes (\).

Both options worked for me in Windows, testing in Command Prompt, Node.js Command Prompt and Git Bash.

Which approach should we take? And if we go for the double-escaped backslashes, how can we do it neatly in the code?

If someone can point me to the right direction I can submit a PR.

**Comment by GAntoine** [comment]
Judging by the docs and what you posted, if Windows does indeed support forward slashes, it would be cleaner to simply use them. If you can open a PR with the changes, that'd be 🔥.

**Comment by jkomyno** [comment]
Same for me

**Comment by kyrylkov** [comment]
@GAntoine so when will you push v2.2.1 to npm?

**Comment by GAntoine** [comment]
@kyrylkov I don't have the permissions to push to npm, but we can ask @marudor.

**Comment by kyrylkov** [comment]
@marudor any update?

**Comment by marudor** [comment]
Sorry, I was busy the last two weeks and didn't notice.
Just released 2.2.1.
